src/game/services/Rules/index.test.ts

```diff
@@ -0,0 +1,207 @@
+import shuffle from 'lodash.shuffle'
+
+import { stubPlayer } from '../../../test-utils/stubs/players'
+import { pumpkin } from '../../cards/crops/pumpkin'
+import { carrot } from '../../cards'
+import {
+  DECK_SIZE,
+  INITIAL_HAND_SIZE,
+  INITIAL_PLAYER_FUNDS,
+  STANDARD_TAX_AMOUNT,
+} from '../../config'
+import { isGame } from '../../types/guards'
+import { handlePlayFromHand as mockCropHandlePlayFromHand } from '../../cards/crops/handlePlayFromHand'
+import { IGame, IPlayer } from '../../types'
+import { updatePlayer } from '../../reducers/update-player'
+import { RandomNumber } from '../../../services/RandomNumber'
+
+import { Rules } from '.'
+
+const player1 = stubPlayer()
+const player2 = stubPlayer()
+
+jest.mock('../../cards/crops/handlePlayFromHand', () => {
+  return {
+    handlePlayFromHand: jest.fn(),
+  }
+})
+
+jest.mock('lodash.shuffle', () => ({
+  __esModule: true,
+  default: jest.fn(),
+}))
+
+beforeEach(() => {
+  ;(shuffle as jest.Mock).mockImplementation((arr: any[]) => arr)
```

Comment:
I tried making this change:

```diff
diff --git a/src/game/services/Rules/index.test.ts b/src/game/services/Rules/index.test.ts
index e8a421d..d1f4bdd 100644
--- a/src/game/services/Rules/index.test.ts
+++ b/src/game/services/Rules/index.test.ts
@@ -28,13 +28,9 @@ jest.mock('../../cards/crops/handlePlayFromHand', () => {
 
 jest.mock('lodash.shuffle', () => ({
   __esModule: true,
-  default: jest.fn(),
+  default: jest.fn((arr: any[]) => arr),
 }))
 
-beforeEach(() => {
-  ;(shuffle as jest.Mock).mockImplementation((arr: any[]) => arr)
-})
-
 // Make player2's deck slightly different from player1's to prevent false
 // positives.
 player2.deck[DECK_SIZE - 1] = pumpkin.id
```

And it led to a number of test failures. I believe this is because the mocked `default` on line 31 persists its mock implementation state across tests. I would have thought that this Jest configuration would have prevented this, but that does not seem to be the case: https://github.com/jeremyckahn/farmhand-shuffle/blob/5af59a937c2d97cd770e3a936d3fdc4bd47354c4/jest.config.js#L6